File: 3830-find-closest-person/find-closest-person.go
func findClosest(x int, y int, z int) int {
	if abs(x-z) < abs(y-z) {
		return 1
	}

	if abs(x-z) == abs(y-z) {
		return 0
	}

	return 2
}

func abs(a int) int {
	if a < 0 {
		return -a
	}

	return a
}

File: 3830-find-closest-person/find-closest-person.java
class Solution {
    public int findClosest(int x, int y, int z) {
        if (Math.abs(x - z) < Math.abs(y - z))
            return 1;

        if (Math.abs(x - z) == Math.abs(y - z))
            return 0;

        return 2;
    }
}